### ISSUE TITLE
some spaces ignored

### ISSUE THREAD

**Issue by TomTasche** [opendocument-app/OpenDocument.core#41]
spaces ignored at the beginning / end of tag. example: https://github.com/TomTasche/OpenDocument.test/blob/b369654970ed7f357cf1f952f0a14945b516e3f1/files/odt/charset%2Bstyle-various-1.odt

**Comment by andiwand** [comment]
i think that was actually not that easy, at least with tinyxml2. we would need to switch the xml parser or modify it for our needs.

**Comment by TomTasche** [comment]
Right, I remember that we talked about this. Changed the estimate to the maximum

**Comment by andiwand** [comment]
starting point https://github.com/andiwand/OpenDocument.core/tree/fix/xml-spacing
